### ISSUE TITLE
GM VISIBLE (BB #636)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#636]
This issue was migrated from bitbucket.
**Original Reporter:** silvermcd123
**Original Date:** 11.08.2014 12:48:20 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** closed
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/636

<hr>

if u use .gm visible off u are only invisible to other gms but not to players

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 11.08.2014 12:52:33 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/636/#comment-11718783

<hr>

hm... do you have gm on ? What revision ?

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 11.08.2014 13:01:19 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/636/#comment-11718951

<hr>

can't confirm

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 11.08.2014 19:58:40 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/636/#comment-11726886

<hr>

Confirmed, thanks!

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 11.08.2014 20:05:34 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/636/#comment-11726985

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 11.08.2014 21:52:40 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/636/#comment-11728405

<hr>

[Core/Players] Add extra check to canSeeOrDetect to fix gm visibility.
               Closes #636.

```
           @todo: We really need to clean up this entire function, it's getting
           giant and complicated and could probably be simplified.
```

→ <<cset 6f075c6ac06e>>
